### PR TITLE
Add security doc for arlas docs

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

File: docs/arlas-wui-security.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  
+  # wuiAuthent.storage -- Defines the kind of storage : localstorage or sessionstorage
+  storage: sessionstorage
+  
+  # wuiAuthent.issuer -- The issuer's uri
+  issuer: CHANGE_ME
+  
+  # wuiAuthent.client_id -- The client's id as registered with the identity provider server
+  client_id: CHANGE_ME
+
```
Comment (MohamedHamouGisaia) [inline docs/arlas-wui-security.md:58]: end with ```

File: docs/arlas-wui-security.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# ARLAS security
+
+This page describes how to configure ARLAS-wui in order to add an access control to the application.
+
+## Authentication
+
+Arlas-wui is compliant with an identity service provider (like https://auth0.com/) which respects an [OAuth 2.0 PKCE : RFC 7636: Proof Key for Code Exchange](tools.ietf.org/html/rfc7636 protocol).
```
Comment (MohamedHamouGisaia) [inline docs/arlas-wui-security.md:7]: (https://tools.ietf.org/html/rfc7636)

File: docs/arlas-wui-security.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# ARLAS security
```
Comment (MohamedHamouGisaia) [inline docs/arlas-wui-security.md:1]: ARLAS-wui security

File: docs/arlas-wui-security.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  # wuiAuthent.require_https -- Defines whether https is required
+  require_https: true
+  
+  # wuiAuthent.response_type -- Response type values
+  response_type: code token id_token
+  
+  # wuiAuthent.silent_refresh_timeout -- Timeout for silent refresh
+  silent_refresh_timeout: "10000"
+  
+  # wuiAuthent.timeout_factor -- Defines when the token_timeout event should be raised. If you set this to the default value 0.75, the event s triggered after 75% of the token's life time.
```
Comment (sebbousquet) [inline docs/arlas-wui-security.md:35]: "the event **is** triggered"

**Review by sebbousquet — APPROVED** [review]

File: docs/arlas-wui-security.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  scope: openid profile
+  
+  # authentication.require_https -- Defines whether https is required
+  require_https: true
+  
+  # authentication.response_type -- Response type values
+  response_type: code token id_token
+  
+  # authentication.silent_refresh_timeout -- Timeout for silent refresh
+  silent_refresh_timeout: "10000"
```
Comment (MohamedHamouGisaia) [inline docs/arlas-wui-security.md:33]: without ""

File: docs/arlas-wui-security.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  
+  # authentication.timeout_factor -- Defines when the token_timeout event should be raised. If you set this to the default value 0.75, the event is triggered after 75% of the token's life time.
+  timeout_factor: "0.75"
```
Comment (MohamedHamouGisaia) [inline docs/arlas-wui-security.md:36]: without ""

**Review by MohamedHamouGisaia — APPROVED** [review]

**Comment by MohamedHamouGisaia** [comment]
in `mkDocs.sh` we should add a line that copies arlas-wui-security.md in generated-docs folder
```
cp docs/arlas-wui-security.md target/generated-docs/arlas-wui-security.md
```

**Pull request merged**